src/components/tabs/tabs-theme.scss

```diff
@@ -97,25 +51,37 @@ md-tabs.md-THEME_NAME-theme {
     }
   }
 
+  &.md-accent, &.md-primary, &.md-warn{
+    > md-tabs-wrapper {
+      > md-tabs-canvas {
+        > md-pagination-wrapper {
+          > md-tab-item:not([disabled]) {
+            color: rgba(255, 255, 255, 0.7);
```

Comment:
This is not theme related, the background of tabs will look the same in dark and light themes,
this is only the not selected tabs labels and it should be 70% white according the spec